apollo-router/src/plugins/telemetry/apollo.rs

```diff
@@ -136,8 +137,8 @@ const fn client_version_header_default() -> HeaderName {
     HeaderName::from_static(client_version_header_default_str())
 }
 
-pub(crate) const fn default_buffer_size() -> usize {
-    10000
+pub(crate) const fn default_buffer_size() -> NonZeroUsize {
+    unsafe { NonZeroUsize::new_unchecked(10000) }
```

Comment:
I really dislike this api, but ...

Reply:
there should be a panicking const constructor for `NonZeroUsize`, but there is not :(